### ISSUE TITLE
Update issue 

### ISSUE THREAD

**Issue by isAAAc** [YunoHost-Apps/piwigo_ynh#62]
Hello, 
i can't update piwigo on 2 different servers, 
you cann find the log here: 

- https://paste.yunohost.org/raw/ixolexuzat
- https://paste.yunohost.org/raw/egusaraduf

```
2021-02-28 17:38:51,337: DEBUG - + ynh_permission_has_user --permission=main --user=visitors
2021-02-28 17:38:51,340: DEBUG - + ynh_permission_exists --permission=main
2021-02-28 17:38:51,442: DEBUG - + yunohost user permission list --short
2021-02-28 17:38:51,442: DEBUG - + grep --word-regexp --quiet piwigo.main
2021-02-28 17:38:51,944: DEBUG - + yunohost user permission info piwigo.main
2021-02-28 17:38:51,944: DEBUG - + grep --word-regexp --quiet visitors
2021-02-28 17:38:52,446: DEBUG - + ynh_exit_properly
2021-02-28 17:38:52,447: DEBUG - + local exit_code=1
2021-02-28 17:38:52,447: DEBUG - + '[' 1 -eq 0 ']'
2021-02-28 17:38:52,447: DEBUG - + trap '' EXIT
2021-02-28 17:38:52,447: DEBUG - + set +o errexit
2021-02-28 17:38:52,448: DEBUG - + set +o nounset
2021-02-28 17:38:52,448: DEBUG - + sleep 0.5
2021-02-28 17:38:52,950: DEBUG - + type -t ynh_clean_setup
2021-02-28 17:38:52,953: WARNING - [Error] Upgrade failed.
2021-02-28 17:38:52,954: DEBUG - + ynh_clean_setup
2021-02-28 17:38:52,954: DEBUG - + ynh_restore_upgradebackup
```
not very verbose about what is happening
any help is welcomed ;)

**Comment by isAAAc** [comment]
nobody is encountering this issue ?

**Comment by isAAAc** [comment]
same error on a third server, 
update of piwigo is completely KO imho

**Comment by isAAAc** [comment]
related to https://github.com/YunoHost-Apps/piwigo_ynh/pull/63

**Comment by ericgaspar** [comment]
@isAAAc the patch is in testing now. if you want to test ...

**Comment by isAAAc** [comment]
@ericgaspar could you refresh my memory about how to try in testing mode please ?

**Comment by ericgaspar** [comment]
It should be `sudo yunohost app upgrade piwigo -u https://github.com/YunoHost-Apps/piwigo_ynh/tree/testing`

**Comment by isAAAc** [comment]
looks OK
```
# yunohost app upgrade piwigo -u https://github.com/YunoHost-Apps/piwigo_ynh/tree/testing
Info: Now upgrading piwigo...
Info: [....................] > Loading installation settings...
Info: [+...................] > Ensuring downward compatibility...
Info: [#+++................] > Backing up Piwigo before upgrading (may take a while)...
Info: [####+...............] > Upgrading source files...
Info: [#####+..............] > Upgrading NGINX web server configuration...
Info: [######+++...........] > Upgrading dependencies...
Info: [#########...........] > Making sure dedicated system user exists...
Info: [#########++.........] > Upgrading PHP-FPM configuration...
Info: [###########++.......] > Upgrading Piwigo with cURL...
Info: [#############+......] > Configuring Piwigo...
Info: [##############......] > Configuring LDAP plugin...
Info: [##############++++..] > Reconfiguring Fail2Ban...
Info: The service fail2ban has correctly executed the action reload-or-restart.
Info: [##################+.] > Configuring permissions...
Info: [###################.] > Reloading NGINX web server...
Info: [####################] > Upgrade of Piwigo completed
Success! piwigo upgraded
Success! Upgrade complete
```
access to the app is OK, 
the app webui tells it is uptodate 

good job <3

**Comment by ericgaspar** [comment]
Fixed with #64
